### ISSUE TITLE
Autocomplete directories if directory is first word

### ISSUE THREAD

**Issue by imoby** [warpdotdev/warp#133]
**Is your feature request related to a problem? Please describe.**
In terminal and iterm, I can open a new editor and the first word can be the name of a directory. For example if I typed in `desk` and then tab. it should autocomplete to Desktop

**Describe the solution you'd like**
if I typed in `desk` and then tab. it should autocomplete to Desktop

**Comment by elviskahoro** [comment]
This work for me as you've described? Tested for zsh and bash.
It may have been added since when you posted this. Can you circle back and see if it's working for you?
If so, this issue can probably be closed. @alokedesai

**Comment by alokedesai** [comment]
I don't think this is fixed actually, we currently suggest executables in the user's path if it's the first word, not directories. Agreed we should support this though--something we will prioritize fixing!

**Comment by elviskahoro** [comment]
@waldekmastykarz
waldekmastykarz: "When using iTerm2 with zsh, you get path completion for files and folders in the current path without having to begin with ./. This is very convenient and would be a great addition to Warp."

zheng: "Just so I understand, is this for when the input box is empty? Currently, it inserts four spaces, but it should bring up local path completions instead.

As of the last release, you should be getting completions for local filepaths when there's a command in the input box!"

waldekmastykarz: "Correct, the idea is that I can cd to a folder by just typing its name, without having to prepend it with ./"

**Comment by brooksmtownsend** [comment]
> Describe the solution you'd like
if I typed in desk and then tab. it should autocomplete to Desktop

In addition to this, shells like `zsh` will handle the case where if you type a subset of the folder and there's no other folder containing that subset, you can use tab autocomplete.

For example, in a home folder you may have
```
Desktop
Documents
Videos
Music
...
```

In this example typing `cd top<TAB>` should result in autofilling `cd Desktop/`, typing `cd docu<TAB>` should correct the spelling on the first character, etc. Would love to see this addition to Warp

**Comment by elviskahoro** [comment]
@brooksmtownsend gonna attach your feedback to:
- #24
